modules/t/coordSystemAdaptor.t

```diff
@@ -235,7 +235,7 @@ my $old_dbID = $cs->dbID;
 is($csa->remove($cs), $cs, 'Adaptor remove() returns the same coordinate system object');
 is_deeply($csa->{_is_sequence_level}, {}, 'Adaptor sequence_level cache updated to reflect the removal');
 is($csa->{_is_default_version}{$old_dbID}, undef, 'Adaptor default_version cache updated to reflect removal');
-lives_ok { $csa->store($cs); }, 'Adaptor store() successful. Coordinate system did not have dbID() or adaptor()';
+lives_ok({ $csa->store($cs); }, 'Adaptor store() successful. Coordinate system did not have dbID() or adaptor()');
```

Comment:
I haven't actually run this but from my experience with Test::Exception, I am 99% sure this will now fail unless you prefix the opening curly brace with 'sub'.